src/test/java/org/broadinstitute/hellbender/engine/filters/ReadFilterPluginUnitTest.java

```diff
@@ -406,6 +408,22 @@ public void testDisabledNonDefaultFilterWithArgsProvided() {
         clp.getPluginDescriptor(GATKReadFilterPluginDescriptor.class);
     }
 
+    @Test
+    public void testDisableDefaultsAndReenableWithDifferentArgs() {
```

Comment:
I will add also a test of disable all the defaults and re-enable with an user filter before it.